docs/package_templates/cmake_package/all/test_package/test_package.cpp

```diff
@@ -4,11 +4,11 @@
 
 
 int main(void) {
-    std::cout << "Create a minimal usage for the target project here." << std::endl;
-    std::cout << "Avoid big examples, bigger than 100 lines" << std::endl;
-    std::cout << "Avoid networking connections." << std::endl;
-    std::cout << "Avoid background apps or servers." << std::endl;
-    std::cout << "The propose is testing the generated artifacts only." << std::endl;
+    std::cout << "Create a minimal usage for the target project here.\n";
```

Comment:
I bet there's a meme for this diff 🤣

Reply:
-_-

Reply:
The Bike Shed.

Actually, you coult convert it to a block comment I think. I totally forgot when I wrote that code.

Reply:
I combined it to a single output statement.